### PR TITLE
Credscan - revert crypto changes

### PR DESCRIPTION
**Pull request by aik-jahoda**
Please don't review yet
- This PR reverts crypto changes introduced in #404 as discussed in https://github.com/dotnet/corefx/pull/43052#issuecomment-800481195
- fix forgotten UriRelativeResolutionTest.cs
- bump system.net.testdata to 1.0.7 required by #404
- Adds more changes introduced in https://github.com/dotnet/corefx/pull/43052

### CODE REVIEW

**Comment by aik-jahoda** [comment]
cc @SamMonoRT, as dotnet#43052 is merged, this PR is ready to merge

File: src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs [hunk 1]
```diff
@@ -662,7 +662,7 @@ public void DigestValue_CRLF()
             
             Assert.Equal("IKbfdK2/DMfXyezCf5QggVCXfk8=", Convert.ToBase64String(digest));
 
-            X509Certificate2 cert = new X509Certificate2(_pkcs12, "PLACEHOLDER");
+            X509Certificate2 cert = new X509Certificate2(_pkcs12, "mono");
```
Comment (SamMonoRT) [inline src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs:665]: While you are editing this - For this and other changes, should we just add the suppression comment now ? All other crypto/hash testing files have an exception approved already // [SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine", Justification="Not a secret.")]

Comment (aik-jahoda) [reply src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs:665]: Best would be cherrypick https://github.com/dotnet/corefx/pull/43055 once we have all exemptions (probably we already have)

**Review by SamMonoRT — APPROVED** [review]

**Comment by aik-jahoda** [comment]
@SamMonoRT I don't plan to add to this PR, you can merge it

**Pull request merged**